server/events/comment_parser.go

```diff
@@ -129,7 +132,7 @@ func (e *CommentParser) Parse(comment string, vcsHost models.VCSHostType) Commen
 	case models.AzureDevops:
 		vcsUser = e.AzureDevopsUser
 	}
-	executableNames := []string{"run", atlantisExecutable, "@" + vcsUser}
```

Comment:
Need to update the comment_parser_test to pass in TriggerWords to the struct used there.

Also need to add tests there for this new functionality. Can use the same pattern as the helpresponse tests:
```go
func TestParse_HelpResponse(t *testing.T) {
	helpComments := []string{
		"run",
		"atlantis",
		"@github-user",
		"atlantis help",
		"atlantis --help",
		"atlantis -h",
		"atlantis help something else",
		"atlantis help plan",
	}
	for _, c := range helpComments {
		r := commentParser.Parse(c, models.Github)
		Equals(t, events.HelpComment, r.CommentResponse)
	}
}

```